xarray/core/weighted.py

```diff
@@ -119,6 +119,19 @@ def _weight_check(w):
         self.obj: T_DataWithCoords = obj
         self.weights: "DataArray" = weights
 
+    def _check_dim(self, dim=Optional[Union[Hashable, Iterable[Hashable]]]):
```

Comment:
that looks like a type hint. Should that have been a colon?
```suggestion
    def _check_dim(self, dim: Optional[Union[Hashable, Iterable[Hashable]]]):
```

Reply:
Right - thanks!